inform7/Internal/Inter/Architecture16Kit/kinds/BasicIO.neptune

```diff
@@ -0,0 +1,18 @@
+new base BASIC_COLOUR_TY {
```

Comment:
This kind is added to the parser, which seems unnecessary. Is there a way to specify that it shouldn't be understandable?